src/app/components/shared/menu/button/button.component.scss

```diff
@@ -13,6 +11,15 @@ button {
   &:focus {
     background-color: var(--theme-highlight-light);
   }
+
+  &:disabled {
+    cursor: not-allowed;
+
+    &:hover,
+    &:focus {
+      background-color: transparent;
+    }
+  }
```

Comment:
this seems like a potentially reusable style, nothing about it needs it to be limited to the scope of a button. Perhaps refactor?

Also: rather than setting the background color to transparent, why not set it to `unset`? That should have the same effect and be semantically clearer. Also (probably no longer relevant) needless use of transparency historically affected page performance.